package.json

```diff
@@ -28,146 +28,150 @@
   },
   "dependencies": {
     "@iarna/word-count": "^1.1.2",
-    "@jupyterlab/services": "^3.2.1",
-    "@manuscripts/assets": "^0.6.1",
-    "@manuscripts/comment-editor": "^0.2.1",
-    "@manuscripts/manuscripts-json-schema": "^1.53.0",
-    "@manuscripts/plugin-persist": "^0.1.2",
-    "@manuscripts/title-editor": "^0.5.2",
-    "@manuscripts/track-changes-plugin": "^0.4.3",
     "astrocite-eutils": "^0.16.4",
-    "axios": "^0.20.0",
-    "blob-util": "^2.0.2",
-    "citeproc": "^2.4.48",
+    "axios": "^1.2.2",
     "codemirror": "^5.58.1",
-    "crypto-hash": "^1.3.0",
-    "dompurify": "^2.2.2",
+    "comlink": "^4.3.0",
+    "csl-json": "^0.1.0",
+    "dompurify": "^2.4.1",
     "grapheme-splitter": "^1.0.4",
-    "lodash-es": "^4.17.11",
+    "history": "^4.10.1",
+    "lodash-es": "^4.17.21",
     "popper.js": "^1.16.1",
-    "pretty-bytes": "^5.4.1",
-    "prosemirror-commands": "~1.1.7",
-    "prosemirror-dev-tools": "~3.0.2",
-    "prosemirror-dropcursor": "~1.3.4",
-    "prosemirror-gapcursor": "~1.1.5",
-    "prosemirror-history": "~1.1.3",
-    "prosemirror-inputrules": "~1.1.3",
-    "prosemirror-keymap": "~1.1.4",
-    "prosemirror-model": "~1.14.0",
-    "prosemirror-schema-list": "~1.1.4",
-    "prosemirror-state": "~1.3.4",
-    "prosemirror-tables": "~1.1.1",
-    "prosemirror-transform": "~1.3.2",
-    "prosemirror-utils": "~0.9.6",
-    "prosemirror-view": "~1.18.2",
-    "react-dnd": "^11.1.3",
-    "react-modal": "^3.11.2",
-    "react-popper": "^1.0.2",
-    "react-select": "^3.1.0",
-    "smoothscroll-polyfill": "^0.4.4",
-    "uuid": "^8.3.1"
+    "pretty-bytes": "^6.0.0",
+    "uuid": "^9.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.11.6",
-    "@babel/plugin-proposal-class-properties": "^7.10.4",
-    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/polyfill": "^7.11.5",
-    "@babel/preset-env": "^7.11.5",
-    "@babel/preset-react": "^7.10.4",
-    "@babel/preset-typescript": "^7.10.4",
-    "@manuscripts/eslint-config": "^0.4.2",
-    "@manuscripts/examples": "^0.0.8",
-    "@manuscripts/library": "^0.6.3",
-    "@manuscripts/manuscript-transform": "^0.62.0",
-    "@manuscripts/rxdb": "^8.2.0-manuscripts1",
-    "@manuscripts/style-guide": "^0.31.27",
-    "@types/codemirror": "^0.0.98",
-    "@types/dompurify": "^2.0.4",
-    "@types/enzyme": "^3.10.7",
+    "@apollo/client": "^3.7.3",
+    "@babel/core": "^7.20.5",
+    "@babel/preset-env": "^7.20.2",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.18.6",
+    "@manuscripts/assets": "^0.6.1",
+    "@manuscripts/comment-editor": "../manuscripts-comment-editor",
```

Comment:
yarn will not install dependence for me like this, @mnatsheh we should use the published version here? or I'm missing something